### ISSUE TITLE
Downloading of image failed

### ISSUE THREAD

**Issue by kennethlimcp** [lvntbkdmr/NotiGoCMS#1]
I created an article with images and it seems to fail the downloading portion. Any clue? :)

```
$ ./NotiGoCMS -verbose
POST https://www.notion.so/api/v3/getRecordValues
downloaded info about 2 versions in 487.450102ms
POST https://www.notion.so/api/v3/getRecordValues
POST https://www.notion.so/api/v3/loadPageChunk
001 '075ad06e-41d2-44d9-a868-05786751088c' : read from cache in 490.049371ms
POST https://www.notion.so/api/v3/getRecordValues
POST https://www.notion.so/api/v3/loadPageChunk
002 'a2ab542f-01f8-4a07-92e4-ee59e6b98757' : read from cache in 4.518198ms
Downloading https://s3-us-west-2.amazonaws.com/secure.notion-static.com/fc5b22dd-690d-46cf-bad4-1c45e66e030e/IMG20201122144306.jpg ... POST https://www.notion.so/api/v3/getSignedFileUrls
Error: status code 400 Bad Request
Body:
{
  "errorId": "71bb6296-2b3f-4cc7-9160-3fbe5be6243a",
  "message": "Invalid input.",
  "name": "ValidationError"
}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x13cd93d]

goroutine 1 [running]:
main.downloadImage(0xc000492f00, 0xc000294960, 0x9e, 0x1, 0x1, 0x0, 0xc000014640, 0x127, 0x4, 0xc0000a5aa0)
	/Users/kennethlimcp/Desktop/NotiGoCMS/notion_import.go:73 +0x7d
main.downloadAndCacheImage(0xc000492f00, 0xc000121080, 0x76, 0xc00021c600, 0x27, 0x40, 0xc000201c00)
	/Users/kennethlimcp/Desktop/NotiGoCMS/notion_import.go:96 +0x334
main.(*Article).processBlocks(0xc000082d80, 0xc00021c600, 0x27, 0x40)
	/Users/kennethlimcp/Desktop/NotiGoCMS/article.go:501 +0x2e5
main.notionPageToArticle(0xc000492f00, 0xc0001cb300, 0x2)
	/Users/kennethlimcp/Desktop/NotiGoCMS/article.go:553 +0x279
main.loadArticles(0xc0000a5f08, 0x178a2c0, 0x14724a0)
	/Users/kennethlimcp/Desktop/NotiGoCMS/articles.go:146 +0x2b3
main.rebuildAll(0xc00019ff08, 0x178a2c0, 0x1ed)
	/Users/kennethlimcp/Desktop/NotiGoCMS/cms.go:38 +0x3d
main.main()
	/Users/kennethlimcp/Desktop/NotiGoCMS/cms.go:104 +0x345

```

**Comment by kennethlimcp** [comment]
Ah looks like some changes to https://github.com/kjk/notionapi so i did:

```
$ go get -u github.com/kjk/notionapi
```

and pulled in changes from https://github.com/kjk/blog for:
- https://github.com/kjk/blog/blob/master/article.go
- https://github.com/kjk/blog/blob/master/notion_import.go

Thanks!

**Comment by lvntbkdmr** [comment]
Hello @kennethlimcp thanks for the notice. I haven't been updating the repo for 9 months. According to your suggestions I updated the code to adapt newer version of kjk/notionapi. I also added a script named as update_notigocms.sh to be used in accordance with github action to keep the go.mod file updated as the dependency libraries get updated. 
I also changed the GITHUB_TOKEN_OVERRIDE secret variable name to GH_TOKEN_OVERRIDE since the variables starting with GITHUB_ prefix are reserved and cannot be updated or created anymore.
